Description: Disable provisioning
 On Ubuntu, provisioning requires cloud-init. We disable all the
 provisioning functions for the agent.
Author: Ben Howard
Last-Update: 2013-07-11
--- a/config/waagent.conf
+++ b/config/waagent.conf
@@ -1,6 +1,9 @@
 #
 # Windows Azure Linux Agent Configuration
 #
+# Ubuntu uses Cloud-init to provision on Windows Azure. This configuration
+# file is used to ensure that cloud-init does the prep of the disk
+#
 
 # Specified program is invoked with the argument "Ready" when we report ready status
 # to the endpoint server.
@@ -14,29 +17,29 @@
 Role.TopologyConsumer=None
 
 # Enable instance creation
-Provisioning.Enabled=y
+Provisioning.Enabled=n
 
 # Password authentication for root account will be unavailable.
-Provisioning.DeleteRootPassword=y
+Provisioning.DeleteRootPassword=n
 
 # Generate fresh host key pair.
-Provisioning.RegenerateSshHostKeyPair=y
+Provisioning.RegenerateSshHostKeyPair=n
 
 # Supported values are "rsa", "dsa" and "ecdsa".
 Provisioning.SshHostKeyPairType=rsa
 
 # Monitor host name changes and publish changes via DHCP requests.
-Provisioning.MonitorHostName=y
+Provisioning.MonitorHostName=n
 
 # Format if unformatted. If 'n', resource disk will not be mounted.
-ResourceDisk.Format=y
+ResourceDisk.Format=n
 
 # File system on the resource disk
 # Typically ext3 or ext4. FreeBSD images should use 'ufs2' here.
 ResourceDisk.Filesystem=ext4
 
-# ount point for the resource disk
-ResourceDisk.MountPoint=/mnt/resource
+# Mount point for the resource disk
+ResourceDisk.MountPoint=/mnt
 
 # Create and use swapfile on resource disk.
 ResourceDisk.EnableSwap=n
